std/os/linux.zig

```diff
@@ -1534,6 +1535,68 @@ pub const dirent64 = extern struct {
     d_name: u8, // field address is the address of first byte of name https://github.com/ziglang/zig/issues/173
 };
 
+pub const dl_phdr_info = extern struct {
+    dlpi_addr: usize,
+    dlpi_name: [*c]const u8,
+    dlpi_phdr: [*c]*c_void,
+    dlpi_phnum: u16,
+};
+
+const DynLib = @import("../dynamic_library.zig");
+
+// XXX: This should be weak
```

Comment:
Is this PR ok to merge even before we solve the weak thing? What are the implications of this?